flake.nix

```diff
@@ -6,31 +6,47 @@
     nixpkgs.follows = "haskell-backend/nixpkgs";
     llvm-backend.inputs.nixpkgs.follows = "haskell-backend/nixpkgs";
     flake-utils.url = "github:numtide/flake-utils";
-    mavenix.url = "github:nix-community/mavenix";
+    mavenix.url = "github:goodlyrottenapple/mavenix";
```

Comment:
Why use a fork of `nix-community/mavenix` ?

Reply:
See the diff here https://github.com/nix-community/mavenix/compare/master...goodlyrottenapple:mavenix:master
Basically I added some tweaks to `buildMaven` to stop the resulting derivation copying some jars we depend on. Instead we soft link to them from the mavenix generated repo derivation. See here:

https://github.com/runtimeverification/k/blob/1dfda423272bd1ddf9067382e153ace741ce7dcf/nix/k.nix#L39-L46

Reply:
If we're going to depend on this change, even if it's a small one we probably want the code to live in the RV org - @F-WRunTime @ehildenb?